src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,29 @@
+import { FC } from 'react';
+import 'styles/app.scss';
+import MenuFilters from './MenuFilters';
+import MenuItem from './MenuItem';
+
+interface MenuProps {
+  menu: Array<string>;
+}
+
+const Menu: FC<MenuProps> = ({ menu }) => {
+  const menuItems = Array.from(menu).map((_value) => ({
```

Comment:
This does not work
`crypto.randomUUID()` is added to the component. This means that every time component rerenders, new `menuItems` variable creates and thus new ids are assigned.
To make `menuItems` constant, we can define it outside of the react component.

Reply:
Let's pretend for now we already receive `menu` from backend with ids